taskfiles/k8s.yml

```diff
@@ -310,3 +318,16 @@ tasks:
     cmds:
       - 'echo "~~~ Creating operator v2 helm e2e tests :k8s:"'
       - ./hack/v2-helm-setup.sh
+
+  package-and-publish-operator-chart:
+    cmds:
+      - defer: 'helm registry logout registry-1.docker.io'
+      - helm registry login registry-1.docker.io -u {{.DOCKERHUB_USER}} --password {{.DOCKERHUB_TOKEN}}
+      - helm dep update charts/operator
+      # To change name of the operator helm chart malformer program is executed
+      - go run ./nightly-malformer/...
+      - helm package charts/operator
+      - helm push `ls redpanda-operator-nightly*.tgz` oci://registry-1.docker.io/redpandadata
```

Comment:
Does helm push extract the tag from `version`?

Reply:
Loader loads helm chart metadata https://github.com/helm/helm/blob/872198b610db230c69a37bed5ab7b0c00ccfe389/pkg/pusher/ocipusher.go#L60
then it generates path/uri/reference https://github.com/helm/helm/blob/872198b610db230c69a37bed5ab7b0c00ccfe389/pkg/pusher/ocipusher.go#L89-L91 to push to registry. Rest is done by the docker hub.